com.beeper.Beeper.yaml

```diff
@@ -0,0 +1,82 @@
+app-id: com.beeper.Beeper
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: startbeeper
+separate-locales: false
+finish-args:
+  # Version required to use the document-portal v4
+  - --require-version=1.7.0
+  # Required due to being a GUI application
+  - --socket=x11
+  # Required to make sure x11 performance is achived on all platforms
+  # At least that's what the legends tell. it might be worth experimenting
+  # with dropping this permission.
+  - --share=ipc
+  # Required for experimental wayland support
+  - --socket=wayland
+  # Required to provide Call functionality
+  - --socket=pulseaudio
+  - --device=all
+  # As a chat application, networking is required
+  - --share=network
+  # Required for notifications in various desktop environments
+  - --talk-name=org.freedesktop.Notifications
```

Comment:
If you upgrade to 23.08, drop this.

Reply:
When I drop the network one it has no internet :P

Reply:
Notifications talk (test if it works without)

Reply:
Works :)